libs/intro/README.md

```diff
@@ -0,0 +1,7 @@
+# intro
+
+This library was generated with [Nx](https://nx.dev).
```

Comment:
add a readme?

Reply:
I'm not sure it will be published to npm
I think we can remove readme at all